toolchain/driver/clang_runner.h

```diff
@@ -61,12 +97,47 @@ class ClangRunner : ToolRunnerBase {
   // behavior.
   auto EnableLeakingMemory() -> void { enable_leaking_ = true; }
 
+  // Set a pre-built target runtimes directory to use for target-specific
+  // invocations.
+  //
+  // This is provided as a method so that clients can use
+  // `BuildTargetResourceDir` to construct the desired runtimes directory first,
+  // and then set it here for subsequent use. Note that setting this disables
+  // any on-demand building of runtimes.
+  auto SetTargetRuntimesDir(const std::filesystem::path& target_runtimes_path)
+      -> void;
```

Comment:
Is this left over from a previous iteration? Looks like it's not defined or used.

Reply:
Oops, yes. Removed.